src/BaGet/Program.cs

```diff
@@ -4,63 +4,79 @@
+                app.Command("import", import =>
                 {
-                    downloads.OnExecuteAsync(async cancellationToken =>
+                    import.Command("downloads", downloads =>
                     {
-                        var host = CreateHostBuilder(args).Build();
-                        var importer = host.Services.GetRequiredService<DownloadsImporter>();
-
-                        await importer.ImportAsync(cancellationToken);
+                        downloads.OnExecuteAsync(async cancellationToken =>
+                        {
+                            var host = CreateHostBuilder(args).Build();
+                            var importer = host.Services.GetRequiredService<DownloadsImporter>();
+                            await importer.ImportAsync(cancellationToken);
+                        });
                     });
                 });
-            });
-
-            app.OnExecuteAsync(async cancellationToken =>
-            {
-                var host = CreateWebHostBuilder(args).Build();
 
-                await host.RunMigrationsAsync(cancellationToken);
-                await host.RunAsync(cancellationToken);
-            });
+                app.OnExecuteAsync(async cancellationToken =>
+                {
+                    var host = CreateWebHostBuilder(args).Build();
+                    await host.RunMigrationsAsync(cancellationToken);
+                    await host.RunAsync(cancellationToken);
+                });
 
-            await app.ExecuteAsync(args);
-        }
+                await app.ExecuteAsync(args);
+            }
 
-        public static IHostBuilder CreateWebHostBuilder(string[] args) =>
-            CreateHostBuilder(args)
-                .ConfigureWebHostDefaults(web =>
-                {
-                    web.ConfigureKestrel(options =>
+            public static IHostBuilder CreateWebHostBuilder(string[] args) =>
+                CreateHostBuilder(args)
+                    .ConfigureWebHostDefaults(web =>
                     {
-                        // Remove the upload limit from Kestrel. If needed, an upload limit can
-                        // be enforced by a reverse proxy server, like IIS.
-                        options.Limits.MaxRequestBodySize = null;
-                    });
+                        web.ConfigureKestrel(options =>
+                        {
+                            // Remove the upload limit from Kestrel. If needed, an upload limit can
+                            // be enforced by a reverse proxy server, like IIS.
+                            options.Limits.MaxRequestBodySize = null;
+                        });
 
-                    web.UseStartup<Startup>();
-                });
+                        var config = new ConfigurationBuilder()
+                            .SetBasePath(Directory.GetCurrentDirectory())
+                            .AddJsonFile("appsettings.json", true)
```

Comment:
Please add a named parameter for the boolean literal:

```suggestion
                            .AddJsonFile("appsettings.json", optional: true)
```

Reply:
Is updated :)